rudder-server/SOURCES/rudder-jetty.service

```diff
@@ -13,6 +13,7 @@ LimitNOFILE=64000
 ExecStart=/opt/rudder/bin/rudder-jetty.sh start
 ExecStop=/opt/rudder/bin/rudder-jetty.sh stop
 SuccessExitStatus=143
+TimeoutStartSec=120s
```

Comment:
@ncharles doesn't this also requires a change on the jetty side?

Reply:
yes, the taimeout within jetty needs to be increased as well - both apply :(

Reply:
Like adding a default value of JETTY_START_TIMEOUT=120 in /etc/default/rudder-jetty